version4.go

```diff
@@ -18,6 +23,8 @@ func New() UUID {
 // NewString is equivalent to the expression
 //
 //    uuid.New().String()
+//
+// Deprecated: Use *Version4.NewString() instead.
```

Comment:
This is not deprecated.  New and NewString are the two most common ways to create a UUID.  Version 4 is by far the most common type of UUID due to the issues associated with Version 1 (which are documented in the RFC).